nixos/modules/i18n/input-method/default.xml

```diff
@@ -60,8 +60,8 @@
 
 <programlisting>
```

Comment:
Yeah, there is a lot of questions. There is actually a third one – detecting whether the code block is a NixOS module. I mainly wanted to add the language attribute to solve that one, since the presence of options unambiguously makes it a nixos module.

We can always resolve the other questions later (I was thinking using jsNix in the browser.)